meetingbaas_pipecat/bot/bot.py

```
@@ -5,20 +5,20 @@
 
 import aiohttp
 import pytz
+import requests
+from deepgram.options import LiveOptions
```

Ruff (0.8.2)

`deepgram.options.LiveOptions` imported but unused

Remove unused import: `deepgram.options.LiveOptions`

(F401)